static_pool.go

```diff
@@ -36,13 +36,13 @@ type StaticPool struct {
 	numDead int64
 
 	// protects state of worker list, does not affect allocation
-	muw sync.RWMutex
+	muw *sync.RWMutex
```

Comment:
what's this for? restored?

Reply:
Actually, the struct functions are using by pointer now, but this will not be necessary in the future for 2.0. Because if smb forgot to use pointer receiver mutex will be copied, which is not good. The pointer in struct solves this problem.